msg/MotionPlanRequest.msg

```diff
@@ -20,6 +20,9 @@ Constraints path_constraints
 # The constraints the resulting trajectory must satisfy
 TrajectoryConstraints trajectory_constraints
 
+# A set of trajectories to be used as reference or seed trajectories for planners
+GenericTrajectory[] reference_trajectories
```

Comment:
Document how usage of this new ``reference_trajectories`` affects the previous approach to goal setting via ``Constraints[] goal_constraints``. Does it replace it? Supplement it?

Reply:
In my opinion they shouldn't replace them, I've amended the comment with this.